project-docs/stumptown-writers-guide.md

```diff
@@ -0,0 +1,697 @@
+
+A chapter list is a YAML file with the following contents:
+
+-   `title`: a mandatory property whose value is a string. This provides a title for the list.
+-   `chapters`: an array of paths to pages to include, relative to the location of the chapter list itself.
+
+For example, the chapter list for HTML guides, which is at content/html/guides:
+
+```
+title: Guides
+chapters:
+    - Block-level_elements
+    - Inline_elements
+    - Link_types
+    - Applying_color
+    - Date_and_time_formats
+    - Preloading_content
+    - CORS_settings_attributes
+    - CORS_enabled_image
+    - DASH_Adaptive_Streaming_for_HTML_5_Video
+    - Microdata
+    - microformats
+    - Quirks_Mode_and_Standards_Mode
+```
+
+## Specifications
+
+Writers use the `specifications` front matter property to describe the specifications that define a particular item. But they just list URLs there, and we want the rendered output to include the name of the specification too.
+
+We could ask writers to include the specification name as well, but this would involve repetition and place a burden on the writers to get the name right. So instead we keep a separate file that maps specification URLs to specification names. This file is called "specifications.yaml" and is kept under content/data.
+
+Here's an example:
+
+```yaml
+# Allowed specification domains and their titles
+# Never add historical specfications to this allow list
+
+html.spec.whatwg.org: 'WHATWG HTML Living Standard'
+w3c.github.io/webappsec-cspee: 'Content Security Policy: Embedded Enforcement'
+w3c.github.io/webappsec-feature-policy: 'Feature Policy'
+wicg.github.io/priority-hints: 'Priority Hints'
+```
+
+Items listed in the `specifications` front matter property must match one of the URLs in this file. They may add extra path elements and fragments, to point to individual items within a specification:
+
+```
+specifications: https://html.spec.whatwg.org/multipage/semantics.html#the-abbr-element
+```
+
+**Note: I don't see a reason why the front matter property includes a scheme, but the scheme is not present in "specifications.yaml".**
+
+## Contributors
+
+Markdown files in stumptown that can be built into pages must have a "contributors.md" file in the same directory. The content of this file is a comma-separated list of Markdown links to MDN user profiles, one for each MDN use who contributed to this page:
+
+```
+[mfuji09](/en-US/profiles/mfuji09),
+[jswisher](/en-US/profiles/jswisher),
+[vadimhp](/en-US/profiles/vadimhp),
+[SphinxKnight](/en-US/profiles/SphinxKnight)
+```
+
+**Note: this is the most hacky and least acceptable piece of stumptown. In particular, I think that legacy contributor information should not live in /content, because it is read-only: authors should never need to be aware of it. But this is only one of its problems.**
```

Comment:
I agree! Maybe it should scrape into /legacy/* with a bunch of other read-only things we will need to have to preserve history?